README.md

```diff
@@ -2,44 +2,37 @@
 
 ## Package updates
 
-How to update the package from Arch on any system
+How to update the package from Arch on any system:
 
-0. Download the release tar.gz and run sha256sum on it.
-1. Edit PKGBUILD and change the `pkgver` and `sha256sums` fields.
-2. Run `mksrcinfo` (see below if you're not on Arch)
-3. Commit & push
+1. Edit the PKGBUILD and change the `pkgver` field.
+2. If you're on Arch, run `updpkgsums && makepkg --printsrcinfo > .SRCINFO`, and skip to the last step.
+3. Download the release tar.gz and run sha256sum on it.
+4. Update the `sha256sums` field in the PKGBUILD.
```

Comment:
Is there no tool to do this for us? Maybe `updpkgsums`?

Reply:
Yeah, that's in step 2, if you're on Arch. Otherwise `updpkgsums` should just be added to the Docker script/Vagrant instructions; I'll do that a little later when I'm back at a computer.

Reply:
Updated.